paddle/fluid/framework/ir/mkldnn/reshape_transpose_matmul_mkldnn_fuse_pass.cc

```diff
@@ -23,12 +23,14 @@ namespace ir {
 
 void ReshapeTransposeMatmulMkldnnFusePass::ApplyImpl(Graph *graph) const {
   auto matmul_types = {"matmul", "matmul_v2"};
+  bool with_reshape_xshape = true;
+  bool with_transpose_xshape = true;
 
   for (const auto &matmul_type : matmul_types) {
-    Fuse(graph, matmul_type, false, false);
-    Fuse(graph, matmul_type, false, true);
-    Fuse(graph, matmul_type, true, false);
-    Fuse(graph, matmul_type, true, true);
+    Fuse(graph, matmul_type, with_reshape_xshape, with_transpose_xshape);
+    Fuse(graph, matmul_type, with_reshape_xshape, !with_transpose_xshape);
+    Fuse(graph, matmul_type, !with_reshape_xshape, with_transpose_xshape);
+    Fuse(graph, matmul_type, !with_reshape_xshape, !with_transpose_xshape);
```

Comment:
```suggestion
    Fuse(graph, matmul_type, reshape_xshape, transpose_xshape);
    Fuse(graph, matmul_type, reshape_xshape, no_transpose_xshape);
    Fuse(graph, matmul_type, no_reshape_xshape, transpose_xshape);
    Fuse(graph, matmul_type, no_with_reshape_xshape, no_transpose_xshape);
```

Reply:
I think that instead of creating 4 variables I will just add inline comments:
`Fuse(graph, matmul_type, false /* with reshape_xshape */, true /* with_transpose_xshape */);`
`Fuse(graph, matmul_type, true /* with reshape_xshape */, false /* with_transpose_xshape */);`

To not restart CI these changes will be added to second PR, which is suplementary to matmul passes cleanup